app/controllers/comments_controller.rb

```diff
@@ -9,15 +9,27 @@ def create
     @comments.post_id = @current_post.id
     @comments.author_id = current_user.id
     if @comments.save
-      flash[:success] = 'Comment created successfully'
-      redirect_to "/users/#{@current_post.author.id}/posts/#{@current_post.id}"
+      redirect_to user_posts_path(@current_post.author_id, @current_post.id), notice: 'New comment saved successfully'
 
     else
-      flash[:error] = 'Something went wrong'
-      render :new, status: :unprocessable_entity
+      render :new, status: :unprocessable_entity, alert: 'Something went wrong'
     end
   end
```

Comment:
- [ ] A user has to be able to add a comment and be able to delete the comment to other users post, kindly implement this feature as it is a reqiurement
![image](https://user-images.githubusercontent.com/81932478/207954813-5f5dbe19-84e8-46f9-ba42-a9a85eb92ebb.png)

Reply:
This change has been done well